ide/refactoring.api/src/org/netbeans/modules/refactoring/spi/ui/ScopeProvider.java

```diff
@@ -72,7 +71,6 @@ public abstract class ScopeProvider {
      *
      * @see ScopeProvider#initialize(org.openide.util.Lookup,
      * java.util.concurrent.atomic.AtomicBoolean)
-     * @see ScopeDescription
      * @return an icon for the Scope, or null if the default should be used.
```

Comment:
this class was never implemented. 
was intended first https://bz.apache.org/netbeans/show_bug.cgi?id=217347#a0 (see attachement).

https://github.com/apache/netbeans/blob/master/ide/refactoring.api/src/org/netbeans/modules/refactoring/spi/impl/ScopeAnnotationProcessor.java could be fixed to remove the SupportedAnnotationTypes ScopeDescription